hack/ginkgo-e2e.sh

```diff
@@ -56,6 +59,7 @@ else
     prepare-e2e
 
     detect-master >/dev/null
+    KUBE_MASTER_URL="https://${KUBE_MASTER_IP:-}" 
```

Comment:
I believe you want
`KUBE_MASTER_URL="${KUBE_MASTER_URL:-https://${KUBE_MASTER_IP:-}"`

use the var if it's set, default to https://IP otherwise.